frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-status-indicator-chip/common-status-indicator-chip.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-chip class="status-indicator" [ngClass]="chipClass">
+  {{ convertKebabToTitleCase(chipClass) }}
```

Comment:
functions in templates are generally not best practice because they will run each time change-detection happens. that really doesn't matter here, but still would be best to just run this function in the constructor or ngOnInit instead of in the template.